models/action.go

```diff
@@ -431,10 +438,13 @@ func (pc *PushCommits) ToAPIPayloadCommits(repoLink string) []*api.PayloadCommit
 				Email:    commit.CommitterEmail,
 				UserName: committerUsername,
 			},
+			Added:     fileStatus.Added,
```

Comment:
Can you add tests to see if `added`, `removed` and `modified` show up with correct information?

Reply:
done.